odo/backends/tests/test_sql.py

```diff
@@ -279,6 +292,28 @@ def test_discover_oracle_intervals(freq):
     assert discover(t) == dshape('var * {dur: ?timedelta[unit="%s"]}' % freq)
 
 
+@pytest.mark.parametrize(
+    'typ,dtype', (
+        (sa.DATETIME, datetime_),
+        (sa.TIMESTAMP, datetime_),
+        (sa.FLOAT, float64),
+        (sa.DATE, date_),
+        (sa.BIGINT, int64),
+        (sa.INTEGER, int_),
+        (sa.BIGINT, int64),
+        (sa.types.NullType, string),
+        (sa.REAL, float32),
+        (sa.Float, float64),
+        # sa.Float(precision=24), float32 (reason="Currently returns float64")
```

Comment:
looks like this should be a skip

Reply:
Yes, now that https://github.com/blaze/odo/pull/590 is merged, I'll use `pytest.param` to mark this as an `xfail`.

Reply:
https://github.com/blaze/odo/pull/589/commits/8e9f3c9d85ab2850bcb77870538bb7b493d1abc2